### PR TITLE
fix: update readme

### PR DESCRIPTION
**Pull request by Unerty**
fix: update readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -212,7 +212,7 @@ See [AccountProps](https://github.com/pepperize/cdk-organizations/blob/main/API.
 
 ### Delegated Administrator
 
-To enable trusted access for a supported AWS service (trusted service), which performs tasks in your organization and its accounts on your behalf, call `delegateAdministrator` on your organization:
+A compatible AWS service can register an AWS member account in the organization as an administrator for the organization's accounts in that service, using `delegateAdministrator` on your account:
```
Comment (pflorek) [inline README.md:215]:
```suggestion
A compatible AWS service (trusted service) can register an AWS member account in the organization as an administrator in the organization on your behalf. To enable an AWS account as administrator of that trusted  in your organization call `delegateAdministrator` on your account:
```

File: README.md [hunk 2]
```diff
@@ -222,8 +222,9 @@ const account = new Account(stack, "Account", {
 account.delegateAdministrator("stacksets.amazonaws.com");
 ```
 
-- To enable trusted access, you must have [all features](https://github.com/pepperize/cdk-organizations/blob/main/API.md#@pepperize/cdk-organizations.FeatureSet) enabled.
-- [AWS services that you can use with AWS Organizations](https://docs.aws.amazon.com/organizations/latest/userguide/orgs_integrate_services_list.html)
+- [AWS services that support Delegated Administrator](https://docs.aws.amazon.com/organizations/latest/userguide/orgs_integrate_services_list.html)
+- To be able to use Delegated Administrator, your organization must have all [all features](https://github.com/pepperize/cdk-organizations/blob/main/API.md#@pepperize/cdk-organizations.FeatureSet) enabled.
+- You must have enabled trusted access for Management Account in your organization.
```
Comment (pflorek) [inline README.md:227]:
```suggestion
- To be able to use Delegated Administrator, your organization must have all [all features](https://github.com/pepperize/cdk-organizations/blob/main/API.md#@pepperize/cdk-organizations.FeatureSet) enabled.
```

**Review by pflorek — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Unerty) [reply README.md:227]: I'd haven't deleted the line You must have enabled trusted access for Management Account in your organization.https://docs.aws.amazon.com/accounts/latest/reference/using-orgs-delegated-admin.html

Comment (pflorek) [reply README.md:227]: Explain how to enable access for account management, i.e.:

> You must have enabled trusted access for Management Account in your organization: 
>  ```typescript
>  organization.enableAwsServiceAccess("account.amazonaws.com")
>  ```
>  See [Enable trusted access](https://github.com/pepperize/cdk-organizations#enable-an-aws-service-trusted-service)

**Review by pflorek — APPROVED** [review]

**Pull request merged**